srcpkgs/python3-kaitaistruct/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-kaitaistruct'
+pkgname=python3-kaitaistruct
+version=0.9
+revision=1
+wrksrc="kaitai_struct_python_runtime-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3"
+short_desc="Kaitai Struct API for Python"
+maintainer="Evgeny Ermakov <evgeny.v.ermakov@gmail.com>"
+license="MIT"
+homepage="https://kaitai.io"
+distfiles="https://github.com/kaitai-io/kaitai_struct_python_runtime/archive/${version}.tar.gz"
+checksum=d21207d17603d87f263c8adcd45235ce22d154ba6faba0945e4f1a9d8d95e7e9
+make_check=no  # Package ships no tests
```

Comment:
this shouldn't be an issue, does the build fail when no tests are ran?

Reply:
Honestly I don't even remember why I added this line.

Reply:
Fixed.